backend/core/config.py

```
@@ -3,36 +3,78 @@
 
 # FrontEnd URL
-FRONTEND_URL = os.getenv("FRONTEND_URL")
-print(f"frontend url:{FRONTEND_URL}")
+FRONTEND_URL = _clean_env("FRONTEND_URL")
 
 # Backend URL
-BACKEND_URL = os.getenv("BACKEND_URL")
-REDIS_URL = os.getenv("REDIS_URL")
+BACKEND_URL = _clean_env("BACKEND_URL")
+REDIS_URL = _clean_env("REDIS_URL") or "redis://localhost:6379/0"
+REDIS_SSL_VERIFY = _get_bool_env("REDIS_SSL_VERIFY", False)
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)